### PR TITLE
CV styled

### PR DESCRIPTION
**Pull request by seand52**

### CODE REVIEW

File: staff/sean-daryanani/cv-styled-sass/index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                <ul class="projects__list">
+                    <li><b>Pasapalabra Game</b> - This game is was made using HTML, CSS, and Javascript</li>
+                    <li><b>Calculator</b> - Web-based calculator that uses HTML, CSS, and Javascript</li>
+                </ul>
+            </section>
+
+            <section class="work">
+                <h2 class="work__title">Work Experience</h2>
+               
+                    <section class="work__description">
```
Comment (manuelbarzi) [inline staff/sean-daryanani/cv-styled-sass/index.html:73]: may work__description be another block-element by itself? then in that case you would have two block elements:

- work
- work-description

each one with its sub-classes (sub-elements and modifiers)
and may work-description be nest-able inside work

File: staff/sean-daryanani/cv-styled-sass/index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                        <h4 class="work__description__title">Analytics Consultant at KAE</h4>
```
Comment (aleixcam) [inline staff/sean-daryanani/cv-styled-sass/index.html]: Title is an element inside the description block, for what I understand the classes should be ```description```and ```description__title``` respectively. Same changes for the ones below.

May @manuelbarzi want to clarify

File: staff/sean-daryanani/cv-styled-sass/index.html (hunk 1 above)
Comment (manuelbarzi) [reply staff/sean-daryanani/cv-styled-sass/index.html:73]: SEE https://medium.com/fed-or-dead/battling-bem-5-common-problems-and-how-to-avoid-them-5bbd23dee319

File: staff/sean-daryanani/cv-styled-sass/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
+%section-titles {
+    width: 100%;
+    background: rgb(98, 195, 233);
+    color: white;
+    border-radius: 10px;
+    text-align: center;
+    @media only screen and (min-width: 768px) {
+        border-top-left-radius: 0px;
```
Comment (JudithGo) [inline staff/sean-daryanani/cv-styled-sass/_variables.scss]: border-radius: 0px 10px 10px 0px;

File: staff/sean-daryanani/cv-styled-sass/index.html (hunk 2 above)
Comment (seand52) [reply staff/sean-daryanani/cv-styled-sass/index.html]: Thanks, you are right!

File: staff/sean-daryanani/cv-styled-sass/index.html (hunk 1 above)
Comment (seand52) [reply staff/sean-daryanani/cv-styled-sass/index.html:73]: Thanks, changed it

**Pull request merged**